src/component/common/EnvironmentStrategiesDialog/EnvironmentStrategyDialog.tsx

```diff
@@ -29,6 +29,7 @@ const EnvironmentStrategyDialog = ({
             onClose={() => onClose()}
             title="You need to add a strategy to your toggle"
             primaryButtonText="Take me directly to add strategy"
+            permission={UPDATE_FEATURE}
```

Comment:
This is not the correct permission. The permission here is UPDATE_FEATURE_STRATEGY, and it needs to take into account the projectId and the environment for that permission.